src/OSPSuite.Core/Domain/Data/DataInfo.cs

```diff
@@ -61,12 +61,6 @@ public class DataInfo
       /// </summary>
       public DateTime Date { get; set; }
```

Comment:
Why do we need the Date? I would remove this too. Why do we care when this column was imported? (in the project there is a history and there is also the configuration)